xbmc/cores/AudioEngine/Engines/ActiveAE/ActiveAEStream.cpp

```diff
@@ -584,6 +585,7 @@ CActiveAEStreamBuffers::CActiveAEStreamBuffers(AEAudioFormat inputFormat, AEAudi
   m_inputFormat = inputFormat;
   m_resampleBuffers = new CActiveAEBufferPoolResample(inputFormat, outputFormat, quality);
   m_atempoBuffers = new CActiveAEBufferPoolAtempo(outputFormat);
+  m_atempoThreshold = CSettings::GetInstance().GetInt(CSettings::SETTING_AUDIOOUTPUT_ATEMPOTHRESHOLD) / 100.0;
```

Comment:
audio settings are managed by AE, loaded by CActiveAE::LoadSettings()
m_atempoThreshold can be set by CActiveAE::CreateStream